src/Common/Exception/CollectionIsEmpty.php

```diff
@@ -10,6 +10,11 @@
 
 namespace Geocoder\Exception;
 
-class CollectionIsEmpty extends \RuntimeException implements Exception
+/**
+ * When you are trying to access an element on en empty collection.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
```

Comment:
Thank you for the review. I know it does not have lots of importance. I like to use this as "Who knows the most of this class". It will make it easier to ping someone if one has questions. Though, yes it is silly in this case. I included it because it is a standard class comment of mine.